cfme/tests/containers/test_smart_management.py

```diff
@@ -58,13 +60,22 @@ def set_random_tag(instance):
     return Tag(display_name=random_tag.text, category=random_cat.text)
 
 
+def wait_for_tag(obj_inst):
+    # Waiting for some tag to appear at "My Company Tags" and return pop'ed last tag
+    try:
+        return wait_for(lambda: getattr(obj_inst.summary.smart_management,
+                                        'my_company_tags', []), fail_condition=[],
+                        num_sec=30, delay=5, fail_func=obj_inst.summary.reload).out.pop()
+    except TimedOutError:
```

Comment:
1) Why did you choose to silence the exception?

2) Returning none will cause problems at lines 89 and 98